tests/integration/test_load_and_run_multi_agent_a2a_tool.py

```diff
@@ -27,19 +27,15 @@
     reason="Integration tests require `ANY_AGENT_INTEGRATION_TESTS=TRUE` env var",
 )
 @pytest.mark.asyncio
-async def test_load_and_run_multi_agent_a2a(
-    agent_framework: AgentFramework,
+async def test_load_and_run_multi_agent(
```

Comment:
Is this now the same test name as a test in the `test_load_and_run_multi_agent` file? I'm worried about a naming overlap

Reply:
AFAICT the suite name is considered to name the test, so it would be `tests/integration/test_load_and_run_multi_agent_a2a_tool.py::test_load_and_run_multi_agent`. But if you prefer I can make it use the `_a2a` prefix again.

Reply:
I filter tests using the `-k 'test_name'` parameter, and this will cause unintended behavior, so I'd prefer if it could be a unique name to make my life easier 😆